### PR TITLE
Entity list - prevent draggable number updates while dragging

### PR DESCRIPTION
**Pull request by dback2**
Using a dragStart and dragEnd callback passed to draggableNumber, block entity property update queries from edit.js during the dragging, and trigger it once dragging ends.

Addresses: https://highfidelity.manuscript.com/f/cases/19919/

Pre-Merge Test Plan:
1. Enter Interface in desktop mode into any non-sandbox domain where you have Edit permissions
2. Open Create and create a Shape entity
3. Verify you can click and drag vector values such as Local Position or Local Dimensions quickly left or right to cause large value changes, that the text value continues to update only in the direction you are dragging, and that upon releasing the mouse the value stays as expected. Verify the property change causes the expected changes on the Shape entity. (The value could change after releasing for vector properties such as Local Rotation or numerical properties such as Parent Joint Index if going beyond limits and needing to wrap around)
4. Perform step 3 again with numerical values such as Clone Lifetime or Density
5. Verify if clicking a numerical or vector value and not moving the mouse before releasing it that the step arrows will hide and it will switch to a state with the text highlighted and editable. Verify you can change the value via keyboard, and upon clicking away from the field or hitting enter it will return to the previous state with the step arrows and the numerical value will still be the same. Verify the property change causes the expected changes on the Shape entity.
6. Verify if clicking the left and right step arrows for a vector or numerical value that the property steps up and down as expected, and that the property change causes the expected changes on the Shape entity.
7. Create a Particle entity
8. Verify you can click and drag numerical values quickly left or right to cause large changes, that the text value continues to update in the direction you are dragging, and that upon releasing the mouse the value stays as expected. Also verify the property causes the expected changes on the Particle entity.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15279/HighFidelity-Beta-PR14435-249ebb2.apk

File: scripts/system/edit.js [hunk 1]
```diff
@@ -2356,6 +2362,7 @@ var PropertiesTool = function (opts) {
                 }
             }
             pushCommandForSelections();
+            nextPropertyUpdateDisabled = data.blockUpdateCallback === true;
             selectionManager._update(false, this);
```
Comment (huffman) [inline scripts/system/edit.js]:
```cpp
  blockPropertyUpdates = data.blockUpdateCallback === true;
  selectionManager._update(false, this);
  blockPropertyUpdates = false;
```
Minor nitpick, but this seems clearer to me than relying on code elsewhere to change it.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15279/HighFidelity-Beta-PR14435-249ebb2.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15279/HighFidelity-Beta-Interface-PR14435-249ebb2.dmg

**Comment by JeffClinton** [comment]
Currently testing

**Comment by JeffClinton** [comment]
Stopped testing

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15280/HighFidelity-Beta-PR14435-dff4e34.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15280/HighFidelity-Beta-PR14435-dff4e34.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15280/HighFidelity-Beta-Interface-PR14435-dff4e34.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15280/HighFidelity-Beta-PR14435-dff4e34.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15280/HighFidelity-Beta-Interface-PR14435-dff4e34.exe

**Comment by QliCknox** [comment]
Currently Testing

**Comment by QliCknox** [comment]
Pass @conklin94122 

Property values update promptly

**Pull request merged**